NEWS.md

```diff
@@ -1,3 +1,7 @@
+### v8.17.0 (2022-07-27)
+
+* Added gRPC server instrumentation
```

Comment:
We'll want to be specific on the client library as there are 2 (well at least two main ones).

Maybe something like:

```suggestion
* Added instrumentation for `grpc-js` server unary, client-streaming, server-streaming and bidi-streaming handlers.
```

Reply:
I think the "native" `grpc` library is deprecated, but sure, no harm in being specific.